run-build

```diff
@@ -101,49 +94,53 @@ class ImageBuildRoot(object):
         #         shutil.rmtree(aptcache_dir)
         # os.makedirs(os.path.join(aptcache_dir), exist_ok=True)
 
-        # Generate a sources list
-        sources_args = {
-            'url': repo,
-            'codename': codename,
-            'components': ' '.join(components),
-            'base': base,
-        }
-        sources = dedent('''\
-        deb {url} {codename} {components}
-        deb http://deb.debian.org/debian {base} main contrib non-free
-        deb http://deb.debian.org/debian {base}-updates main contrib non-free
-        deb http://security.debian.org {base}-security main contrib non-free
-        '''.format(**sources_args))
-
-        sources_path = os.path.join(config['build']['tmpdir'],
-                                    'buildroot-sources.list')
-        with open(sources_path, 'w') as f:
-            f.write(sources)
-
-        log.info('Creating buildroot in %s', self.builddir)
-        setup_script = os.path.join(config['build']['helpersdir'],
-                                    'mmdebstrap-setup')
-        cleanup_script = os.path.join(config['build']['helpersdir'],
-                                      'mmdebstrap-cleanup')
-        buildroot_env = os.environ.copy()
-        buildroot_env.update({
-            'BUILDROOT_KEYRING': keyring,
-        })
-        buildroot_cmd = (
-            'mmdebstrap',
-            '--verbose',
-            '--architectures=' + arch,
-            '--variant=minbase',
-            '--include=' + ','.join(packages),
-            '--setup-hook=' + setup_script,
-            '--customize-hook=' + cleanup_script,
-            # Empty suite is used to prevent mmdebstrap from filtering
-            # the required packages.
-            '',
+        # Pull the ostree. Ideally the eosminbase ostree would be used to
+        # minimize the buildroot, but that's not released and would fail with
+        # --use-production-ostree.
```

Comment:
A shame indeed… Maybe we could change that at some point but I don't think we want to block this change on that. (Another way to look at this would be as an incentive to make the OS as small as possible! But the builder never needs GNOME Shell.)